sample-components-prop-visualizer/src/controllers/Quat.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Button } from "magic-script-components";
+
+export default function QuatController({ applyValue }) {
+  return (
+    <>
+      <Button
+        localPosition={[0.32, 0.45, 0]}
+        onClick={() => applyValue("valueX", -0.1)}
+      >
+        {"< X"}
+      </Button>
+
+      <Button
+        localPosition={[0.4, 0.45, 0]}
+        onClick={() => applyValue("valueX", 0.1)}
+      >
+        {"> X"}
+      </Button>
+
+      <Button
+        localPosition={[0.32, 0.4, 0]}
+        onClick={() => applyValue("valueY", -0.1)}
+      >
+        {"< Y"}
+      </Button>
+
+      <Button
+        localPosition={[0.4, 0.4, 0]}
+        onClick={() => applyValue("valueY", 0.1)}
+      >
+        {"> Y"}
+      </Button>
+
+      <Button
+        localPosition={[0.32, 0.35, 0]}
+        onClick={() => applyValue("valueZ", -0.1)}
+      >
+        {"< Z"}
+      </Button>
+
+      <Button
+        localPosition={[0.4, 0.35, 0]}
+        onClick={() => applyValue("valueZ", 0.1)}
+      >
+        {"> Z"}
+      </Button>
+
+      <Button
+        localPosition={[0.32, 0.3, 0]}
+        onClick={() => applyValue("valueA", -0.1)}
+      >
+        {"< A"}
```

Comment:
shouldn't this be W instead of A?  Isn't W more common?

Reply:
W seems to be the convention in the docs. Will change.